parser/parser_test.go

```diff
@@ -1365,6 +1615,13 @@ func TestParse(t *testing.T) {
 					tt.Fatal(test.DiffMessage("location", actualWithLocation, tc.L))
 				}
 			}
+
+			if tc.M != "" {
+				actualAdornedMacroCalls := convertMacroCallsToString(expression.SourceInfo)
+				if !test.Compare(actualAdornedMacroCalls, tc.M) {
+					tt.Fatal(test.DiffMessage("Macro Calls", actualAdornedMacroCalls, tc.M))
```

Comment:
Stylistically, you're supposed to say something like `Method(Args...): got <something>, wanted: <something else>`

The original code didn't follow this pattern, but that's only because I learned of the style preference a bit late. If you wouldn't mind updating the error messages to be of this format, I would appreciate it.

Reply:
Updated the DiffMessage function and where it is called in parser_test.go to follow this pattern.